src/index.js

```diff
@@ -1,5 +1,22 @@
-const trialDivision = (number) => {
-    
-  }
+const trialDivision = number => {
+	let isPrime = true;
+	if (number <= 1) {
+		isPrime = false;
+	} else if (number % 1 !== 0) {
+		isPrime = false;
+	} else if (number % 2 === 0 && number !== 2) {
+		isPrime = false;
+	} else {
+		const square = Math.round(Math.sqrt(number));
+		let divisor = 3;
+		while (isPrime && divisor <= square) {
```

Comment:
puedes usar un for, esta delimitado el rango en el que vas a iterar.

Reply:
El for me haria recorrer todos los numeros hasta "square" y seria menos optimo, la idea es solo calcular el modulo de numeros impares (ya que los pares los pares estan descartados en la anterior condicion) y solo mientras el numero siga siendo primo o hasta que divisor alcance a "square".

Reply:
Puedes usar `divisor %2 === 0 ? divisor += 1 : divisor+=2` para iniciar 2 y luego únicamente con los impares.